images/miq-app/docker-assets/container-scripts/container-deploy-common.sh

```diff
@@ -235,7 +235,7 @@ echo "== Applying memcached config =="
 
 sed -i~ -E "s/:memcache_server:.*/:memcache_server: ${MEMCACHED_SERVICE_NAME}:11211/gi" "${APP_ROOT}/config/settings.yml"
 
-[ "$?" -ne "0" ] && echo "ERROR: Failed to apply memcached configuration, please check journal or PV logs" && exit 1
+[ "$?" -eq "0" ] && return 0 || echo "ERROR: Failed to apply memcached configuration, please check journal or PV logs" && exit 1
```

Comment:
@bazulay I can't , it will fail without this modification, the old check will return 1 (false) since in most cases the return value for the command will actually be zero. (succeeded)